lms/static/sass/views/_homepage.scss

```diff
@@ -104,7 +104,6 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
           margin: ($baseline*0.25) 0 ($baseline*1.75) 0;
           padding: 0 ($baseline*0.75);
           height: $course-title-height;
-          color: $link-color;
```

Comment:
Why was this taken out? Is it needed or does the reset work now cover this? Have you tested it locally?